File: Programing EF DbContext/StartingSolution/Model/Destination.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Model
{
  [Table("Locations", Schema = "baga")]
  public class Destination
  {
    public Destination()
    {
      this.Lodgings = new List<Lodging>();
    }

    [Column("LocationID")]
    public int DestinationId { get; set; }
    [Required, Column("LocationName")]
    [MaxLength(200)]
    public string Name { get; set; }
    public string Country { get; set; }
    [MaxLength(500)]
    public string Description { get; set; }
    [Column(TypeName = "image")]
    public byte[] Photo { get; set; }
    public string TravelWarnings { get; set; }
    public string ClimateInfo { get; set; }

    public List<Lodging> Lodgings { get; set; }
  }
}

File: Programming EF Code First/chapter6/Chapter6/Chapter 6 VB/DataAnnotations/Model/Destination.vb
Imports System.ComponentModel.DataAnnotations

Namespace Model
  <Table("Locations", Schema := "baga")>
  Public Class Destination
	<Column("LocationID")>
	Public Property DestinationId() As Integer
	<Required, Column("LocationName"), MaxLength(200)>
	Public Property Name() As String
	Public Property Country() As String
	<MaxLength(500)>
	Public Property Description() As String
	<Column(TypeName := "image")>
	Public Property Photo() As Byte()
	Public Property TravelWarnings() As String
	Public Property ClimateInfo() As String

	Public Property Lodgings() As List(Of Lodging)

	Private _todayForecast As String
	<NotMapped>
	Public Property TodayForecast() As String
	  Get
		  Return _todayForecast
	  End Get
	  Set(ByVal value As String)
		  _todayForecast = value
	  End Set
	End Property
  End Class
End Namespace